src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -253,6 +253,7 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
           Failure(regrets)
         case success => success
       }
+      case _ => //nothing
```

Comment:
It's necessary because before, the only other case that was was being handled was when a workflow is still running and has an external id (line 250). If a workflow is still running and has an external id, we need to abort the process (lines 251-254)

Any other cases, for example when a workspace has a successful submission, are not handled. The added line is so that all other statuses of the workflow _are_ handled (even though we don't actually need to do anything).